test: TestV3LeasePrmote

```
--- FAIL: TestV3LeasePrmote (25.74s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
    panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x6949e7]

goroutine 111689 [running]:
panic(0xee8020, 0xc820014110)
    /usr/local/golang/1.6.2/go/src/runtime/panic.go:481 +0x3e6
testing.tRunner.func1(0xc82098de60)
    /usr/local/golang/1.6.2/go/src/testing/testing.go:467 +0x192
panic(0xee8020, 0xc820014110)
    /usr/local/golang/1.6.2/go/src/runtime/panic.go:443 +0x4e9
google.golang.org/grpc.Invoke(0x7f5574783450, 0xc820015078, 0x11589c0, 0x1e, 0xfc8aa0, 0xc82cc6f3c0, 0xfd5620, 0xc82cc71650, 0x0, 0x0, ...)
    /home/runner/workspace/src/github.com/coreos/etcd/gopath/src/google.golang.org/grpc/call.go:149 +0xf27
github.com/coreos/etcd/etcdserver/etcdserverpb.(*leaseClient).LeaseGrant(0xc82d110910, 0x7f5574783450, 0xc820015078, 0xc82cc6f3c0, 0x0, 0x0, 0x0, 0x7f5570be3970, 0x0, 0x0)
    /home/runner/workspace/src/github.com/coreos/etcd/gopath/src/github.com/coreos/etcd/etcdserver/etcdserverpb/rpc.pb.go:2324 +0xec
github.com/coreos/etcd/integration.leaseExist(0xc82098de60, 0xc822cd5fb0, 0x6d1c550e45927302, 0x1)
    /home/runner/workspace/src/github.com/coreos/etcd/gopath/src/github.com/coreos/etcd/integration/v3_lease_test.go:418 +0x11b
github.com/coreos/etcd/integration.TestV3LeasePrmote(0xc82098de60)
    /home/runner/workspace/src/github.com/coreos/etcd/gopath/src/github.com/coreos/etcd/integration/v3_lease_test.go:73 +0x7e1
testing.tRunner(0xc82098de60, 0x16306b0)
    /usr/local/golang/1.6.2/go/src/testing/testing.go:473 +0x98
created by testing.RunTests
    /usr/local/golang/1.6.2/go/src/testing/testing.go:582 +0x892
exit status 2
```

--- a/integration/cluster.go
+++ b/integration/cluster.go
@@ -795,7 +795,15 @@ func (c *ClusterV3) Terminate(t *testing.T) {
 }
 
 func (c *ClusterV3) RandClient() *clientv3.Client {
-	return c.clients[rand.Intn(len(c.clients))]
+	for i := 0; i < 100; i++ {
+		cli := c.clients[rand.Intn(len(c.clients))]
+		if cli.ActiveConnection() == nil {
+			time.Sleep(10 * time.Millisecond)
+			continue
+		}
+		return cli
+	}
+	panic("failed to get a active client")
 }
 
 func (c *ClusterV3) Client(i int) *clientv3.Client {

